EnglishFilter/sections/specific.txt

```diff
@@ -307,6 +307,9 @@ get-bitcoin.net##div[id^="adm-container-"]
 ||forum.lewdweb.net/service_worker.js
 bevru.club##a[href="https://advertise.exe.io"]
 neowin.net##.sidebar-col .ad
+||rollercoin.com/static/img/referral/banners$domain=krypto-trend.de
+krypto-trend.de#?#.my-2 > div.text-center:has(> div.sticky-top > ins[style^="display:inline-block;width:160px;"])
```

Comment:
Wrong section. Do you need this rule?
Why not `krypto-trend.de#?#.row > div.text-center:has(> div.sticky-top > ins.adsbygoogle)`